CourseGrab/Home/HomeViewController.swift

```diff
@@ -76,13 +112,39 @@ class HomeViewController: UITableViewController {
 
 extension HomeViewController {
```

Comment:
All the code in this section would get simplified if we go with my suggestion above about `TableSection` or a similar one.